framework/branch-key-store.md

```diff
@@ -9,6 +9,8 @@
 
 ### Changelog
 
+- 0.7.1
+  - Branch key creation only uses customer input (pre 0.5 behavior)
```

Comment:
Question: The below changes looks like fixes that adds missing parameter thats send to KMS. 

I am trying to understand what does `Branch key creation only uses customer input` and what does `(pre 0.5 behavior)` mean?

Reply:
I think I got what `Branch key creation only uses customer input` means. It mean `Key ARN` is a customer input but not sure about `(pre 0.5 behavior)`

Reply:
I'm looking more at the changes. I think I'll drop it
```suggestion
  - Branch key creation only uses customer input
```

The idea was that looking at the version changes, I thought that this would be in the 0.5 version change (#271). But when looking more at that PR, I think I was less right.